ui/ducks/bridge/dummy-quotes.ts

```diff
@@ -0,0 +1,4005 @@
+              logoURI: 'https://assets.polygon.technology/tokenAssets/eth.svg',
+              chainAgnosticId: null,
+            },
+            destAsset: {
+              chainId: 42161,
+              address: '0x0000000000000000000000000000000000000000',
+              symbol: 'ETH',
+              name: 'Ethereum',
+              decimals: 18,
+              icon: 'https://assets.polygon.technology/tokenAssets/eth.svg',
+              logoURI: 'https://assets.polygon.technology/tokenAssets/eth.svg',
+              chainAgnosticId: null,
+            },
+            srcAmount: '4956250000000000',
+            destAmount: '4852928026153929',
+          },
+        ],
+      },
+      trade: {
+        chainId: 10,
+        to: '0xB90357f2b86dbfD59c3502215d4060f71DF8ca0e',
+        from: '0xc5fe6ef47965741f6f7a4734bf784bf3ae3f2452',
+        value: '0x11c37937e08000',
+        data: '0x3ce33bff000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000011c37937e0800000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000000f736f636b6574416461707465725632000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002a00000000000000000000000003a23f943181408eac424116af7b7790c94cb97a50000000000000000000000003a23f943181408eac424116af7b7790c94cb97a5000000000000000000000000000000000000000000000000000000000000a4b10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000119baee0ab04000000000000000000000000000000000000000000000000000000000000000140000000000000000000000000000000000000000000000000000027ca57357c00000000000000000000000000716a8b9dd056055c84b7a2ba0a016099465a5187000000000000000000000000000000000000000000000000000000000000014800000011c8a6285e000000000000000000000000c5fe6ef47965741f6f7a4734bf784bf3ae3f245200000000000000000000000086ca30bef97fb651b8d866d45503684b90cb331200000000000000000000000000000000000000000000000000119baee0ab0400000000000000000000000000000000000000000000000000000000000000a4b1000000000000000000000000000000000000000000000000000060dcd8031258000000000000000000000000000000000000000000000000001185250b108f80000000000000000000000000000000000000000000000000000001924963f1dd00000000000000000000000000000000000000000000000000112728d1e75e79000000000000000000000000000000000000000000000000000001924963f1dd00000000000000000000000000000000000000000000000000000000000000c40000000000000000000000000000000000000000000000007d6791da46aae617c18c7b5987f2f25e8bc35083c3c973e71bfa0f7bd70088b0558f63bef3e55bc881d14bd276f41690d864e3d0380bfd4ac557b8b9dde896c51c',
+        gasLimit: 414453,
+      },
+      estimatedProcessingTimeInSeconds: 60,
+    },
+  ],
+};
+
+export const DummyQuotesWithApproval = {
```

Comment:
Minor: move this to test/data/bridge so it can be with the other dummy quotes

Reply:
Done in [2655045](https://github.com/MetaMask/metamask-extension/pull/27262/commits/2655045addad45235aa9ce364cf60649ab9ba358)